src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
It seems like you're trying to link a SCSS file directly in HTML. However, browsers do not understand SCSS files. You should compile your SCSS to CSS and link the resulting CSS file instead.